services-camel/src/test/java/edu/unc/lib/dl/services/camel/enhancements/EnhancementRouterIT.java

```diff
@@ -162,14 +179,32 @@ public void testFolderEnhancements() throws Exception {
     }
 
     @Test
-    public void testImageFile() throws Exception {
+    public void nonBinaryNoSourceImages() throws Exception {
+        CollectionObject collObject = repoObjectFactory.createCollectionObject(null);
+        final Map<String, Object> headers = createEvent(false, collObject.getPid(),
+                Cdr.Collection.getURI(), Container.getURI());
+        template.sendBodyAndHeaders("", headers);
+
+        NotifyBuilder notify1 = new NotifyBuilder(cdrEnhancements)
+                .whenCompleted(1)
+                .create();
+
+        boolean result1 = notify1.matches(5l, TimeUnit.SECONDS);
+        assertTrue("Enhancement route not satisfied", result1);
+
+        verify(addSmallThumbnailProcessor, never()).process(any(Exchange.class));
+        verify(addLargeThumbnailProcessor, never()).process(any(Exchange.class));
```

Comment:
Should check `verify(solrIngestProcessor).process(any(Exchange.class));`  too

Reply:
The solr ingest processor should run in this case, if its not then that would be a bug